docs/typescript.mdx

```diff
@@ -199,6 +201,34 @@ const StyledOriginal = styled(Original, {
 ;<StyledOriginal prop1="1" prop2={2} />
 ```
 
+#### Filter props helper
+
+You can use this helper in your project if you want a more concise forward prop syntax
+
+``` ts
+/**
+ * Use `createPropFilter` with shouldForwardProp to filter out props. Will automatically filter out `theme`
+ * since this prop is passed via emotion's context.
+ */
+
+export function createPropFilter<ComponentProps extends {}>() {
```

Comment:
this is sort of a factory-factory now, it doesnt immediately create prop filter, not a big deal but a little bit misleading - especially because of that it would be great to give a more concrete example below it - one involving a styled call